inventory/src/pages/addItem.js

```diff
@@ -204,11 +213,15 @@ export const AddItem = () => {
           min_quantity: false,
           is_opened: false,
         });
-        setAlert('success', 'Added new item', true);
+        setAlert({
+          severity: 'success',
+          message: 'Successfully added new expiry date!',
```

Comment:
Its new item that has expiry date, different from adding a new expiry to existing item?